packages/manager/modules/billing/src/orders/purchaseOrders/edit-purchase/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,18 @@
+{
+  "purchaseOrders_edit_page_title": "Modifier la référence interne",
+  "purchaseOrders_form_edit_internal_reference": "Créer votre référence interne",
+  "purchaseOrders_form_edit_purchase_order": "Créer votre purchase order",
+  "purchaseOrders_form_edit_internal_reference_label": "Nom de la référence interne",
+  "purchaseOrders_form_edit_internal_reference_placeholder": "Référence interne",
+  "purchaseOrders_form_edit_internal_reference_sub_info": "La référence ne doit pas excéder 20 caractères",
+  "purchaseOrders_form_edit_purchase_order_label": "Nom du purchase order",
+  "purchaseOrders_form_edit_purchase_order_placeholder": "Purchase order",
```

Comment:
**suggestion**: If translations are not required, it can be maintained as a constant.

Reply:
it's apply